engine/servermaster/openapi.go

```diff
@@ -35,12 +34,23 @@ const (
 	apiOpVarProjectID = "project_id"
 	// apiOpVarJobID is the key of job id in HTTP API.
 	apiOpVarJobID = "job_id"
-	// apiOpVarJobID is the key of job type in HTTP API.
-	apiOpJobType = "job_type"
-	// apiOpVarJobID is the key of job config in HTTP API.
-	apiOpJobConfig = "job_config"
 )
 
+// APICreateJobRequest defines the json fields when creating a job with OpenAPI
+type APICreateJobRequest struct {
+	JobType   int32  `json:"job_type"`
+	TenantID  string `json:"tenant_id"`
+	ProjectID string `json:"project_id"`
```

Comment:
`TenantID` and `ProjectID` should be always placed in query parameters as discussed before.